apps/system/sys_proc.c

```diff
@@ -12,14 +12,14 @@
 #include "disk.h"
 #include <string.h>
 
-static int app_ino, app_pid;
-static void sys_spawn(int base);
-static int app_spawn(struct proc_request *req);
+static i32 app_ino, app_pid;
```

Comment:
Similar here, maybe u32? But let me know if you have good reasons to use i32. I am just asking, not suggesting.